music_assistant/controllers/cache.py

```diff
@@ -162,9 +163,10 @@ async def set(
         - checksum: optional argument to store with the cache object
         - persistent: if True the cache object will not be deleted when clearing the cache
         """
+        assert self.database is not None
         if not key:
             return
-        if checksum is not None and not isinstance(checksum, str):
+        if checksum is not None:
             checksum = str(checksum)
```

Comment:
yeah, this is fine like this - if some other (not yet fully typed) piece of the existing code now calls this with a non string value, we will simply convert it.